packages/functional-tests/tests/react-conversion/signup.spec.ts

```diff
@@ -93,7 +93,7 @@ test.describe('severity-1 #smoke', () => {
       await confirmSignupCode.fillOutCodeForm(code);
 
       await expect(page).toHaveURL(/connect_another_device/);
-      await expect(page.getByText('You’re signed into Firefox')).toBeVisible();
+      await expect(page.getByText('You’re signed in to Firefox')).toBeVisible();
```

Comment:
Er, how was this passing? 😅

Reply:
Not sure, I can only assume the string changed recently, but I didn't dig into it. I did double check the UI though, and the text is definitely 'in to' not 'into'.